python/torch_mlir/backends/tosa/__init__.py

```diff
@@ -0,0 +1,70 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+from typing import Optional, Sequence, Union
+
+import torch
+import torch_mlir
+
+from ...compiler_utils import run_pipeline_with_repro_report
+
+def compile(model: torch.nn.Module,
+            example_args: Union[torch_mlir._example_arg, Sequence[torch_mlir._example_arg]],
+            use_tracing: bool = False,
+            ignore_traced_shapes = False,
+            backend_legal_ops: Optional[Sequence[str]] = None,
+            backend_custom_ops: Optional[Sequence[str]] = None,
+            verbose: bool = False):
+    """Convert a PyTorch model to TOSA IR in MLIR.
+
+    Args:
+        model: The PyTorch model to convert.
+        example_args: A list of example arguments to use when inferring the
+            shapes of the arguments to `forward` method of the model.
+            A single tensor is treated as a list of a single tensor.
+            A TensorPlaceholder object is also allowed in the place of any
+            Tensor.
+        output_type: The kind of output to produce. See `OutputType` for more
```

Comment:
use **kwargs to forward these to torch_mlir.compile. No need to duplicate all the docs and signature here.